draft-ietf-acme-acme.md

```diff
@@ -1897,7 +1894,7 @@ Content-Language: en
 {
   "type": "urn:ietf:params:acme:error:unauthorized",
   "detail": "No authorization provided for name example.net",
-  "instance": "http://example.com/acme/doc/unauthorized"
+  "instance": "https://example.com/acme/doc/unauthorized"
```

Comment:
I think this `instance` field should be removed instead of updated to use HTTPS ala https://github.com/ietf-wg-acme/acme/pull/355

Reply:
SG, deleted.